eZ/Bundle/EzPublishCoreBundle/Cache/Driver/Redis/RedisIgbinary.php

```diff
@@ -47,7 +47,6 @@ public function storeData($key, $data, $expiration)
     public static function isAvailable()
     {
         return class_exists('Redis', false)
-            && function_exists('igbinary_serialize')
-            && function_exists('igbinary_unserialize');
+            && extension_loaded('igbinary');
```

Comment:
Actually I favour much function_exists over extension_loaded, and youy should, too. function_exists allows for php polyfills to be used in corner cases.
This is confirmed by the history of js: browser version checking was rejected years ago in favour of feature existence checking

Reply:
Thanks for the review @gggeek! I agreed with @adamwojs that we can safely change `function_exists` to `extension_loaded`, but your comment shed some light on it. I can stay with `function_exists` then, WDYT @andrerom, @alongosz?

Reply:
Probably `function_exists` is safer, but in the future if we'll have to add another `function_exists` check then I would prefer extension checking. 
For now both solutions are acceptable.